WalletWasabi.Fluent/ViewModels/Wallets/LoadingViewModel.cs

```diff
@@ -7,14 +7,17 @@
 using WalletWasabi.Blockchain.Blocks;
 using WalletWasabi.Fluent.Extensions;
 using WalletWasabi.Fluent.Helpers;
+using WalletWasabi.Fluent.ViewModels.Navigation;
+using WalletWasabi.Logging;
 using WalletWasabi.Models;
 using WalletWasabi.Wallets;
 
 namespace WalletWasabi.Fluent.ViewModels.Wallets;
 
-public partial class LoadingViewModel : ActivatableViewModel
+[NavigationMetaData(Title = null)]
+public partial class LoadingViewModel : RoutableViewModel
 {
-	private readonly Wallet _wallet;
+	private readonly WalletPageViewModel _nbwsvm;
```

Comment:
If there was a member prefix `m`, I would assume it's Win32 code :D Use a better name, please.

Reply:
This class is heavily refactored in a later PR. That field is actually removed. I'd keep it as-is here in order to avoid merge conflicts.